index.md

```diff
@@ -34,7 +34,7 @@ Carpentries Instructor Training has the following goals:
 Because we have only limited time, some things are beyond the scope of this training. We will not be learning:
 
 - How to program in R or Python, use the Unix shell, Git or SQL, or any of the other topics taught in Data Carpentry, Library Carpentry, or Software Carpentry workshops.
-- How to create your own lessons from scratch (although you will have a good start on the principles behind that sort of work if you are inspired to learn more).
+- How to [create your own lessons from scratch][cldt-curriculum] (although you will have a good start on the principles behind that sort of work if you are inspired to learn more).
```

Comment:
```suggestion
- How to create your own lessons from scratch (although you will have a good start on the principles behind that sort of work if you are inspired to learn more, for example in our [lesson development training][cldt-curriculum]).
```

Since the list is of things we will *not* cover, I think it's more clear if the link is a more explicit reference to another lesson